<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Akshay-Udhane" failures="5" tests="8" name="OPC UA Connection Service Tests" time="85.062" errors="0" timestamp="2025-10-07T11:03:18 IST">
  <testcase classname="tests.OpUaConnectionServiceTest" name="testOpcUaConnectionConnect" time="21.948">
    <failure type="java.lang.AssertionError" message="Expected status 200 but got 500 expected [200] but found [500]">
      <![CDATA[java.lang.AssertionError: Expected status 200 but got 500 expected [200] but found [500]
at base.RealServiceTestBase.verifyResponseStatus(RealServiceTestBase.java:132)
at tests.OpUaConnectionServiceTest.testOpcUaConnectionConnect(OpUaConnectionServiceTest.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- testOpcUaConnectionConnect -->
  <testcase classname="tests.OpUaConnectionServiceTest" name="testOpcUaConnectionErrorHandling" time="0.015"/>
  <testcase classname="tests.OpUaConnectionServiceTest" name="testOpcUaConnectionInit" time="10.041">
    <failure type="java.lang.AssertionError" message="Response is not valid JSON expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Response is not valid JSON expected [true] but found [false]
at base.RealServiceTestBase.verifyJsonResponse(RealServiceTestBase.java:160)
at tests.OpUaConnectionServiceTest.testOpcUaConnectionInit(OpUaConnectionServiceTest.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- testOpcUaConnectionInit -->
  <testcase classname="tests.OpUaConnectionServiceTest" name="testOpcUaConnectionPerformance" time="0.009"/>
  <testcase classname="tests.OpUaConnectionServiceTest" name="testOpcUaConnectionRetry" time="0.004"/>
  <testcase classname="tests.OpUaConnectionServiceTest" name="testOpcUaConnectionSequence" time="20.102">
    <failure type="java.lang.AssertionError" message="Expected status 200 but got 500 expected [200] but found [500]">
      <![CDATA[java.lang.AssertionError: Expected status 200 but got 500 expected [200] but found [500]
at base.RealServiceTestBase.verifyResponseStatus(RealServiceTestBase.java:132)
at tests.OpUaConnectionServiceTest.testOpcUaConnectionSequence(OpUaConnectionServiceTest.java:61)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- testOpcUaConnectionSequence -->
  <testcase classname="tests.OpUaConnectionServiceTest" name="testOpcUaConnectionStatus" time="0.012">
    <failure type="java.lang.AssertionError" message="Response does not contain expected text: Connected expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Response does not contain expected text: Connected expected [true] but found [false]
at base.RealServiceTestBase.verifyResponseContains(RealServiceTestBase.java:146)
at tests.OpUaConnectionServiceTest.testOpcUaConnectionStatus(OpUaConnectionServiceTest.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- testOpcUaConnectionStatus -->
  <testcase classname="tests.OpUaConnectionServiceTest" name="testOpcUaServiceHealth" time="0.006">
    <failure type="java.lang.AssertionError" message="OPC UA service should be healthy expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: OPC UA service should be healthy expected [true] but found [false]
at tests.OpUaConnectionServiceTest.testOpcUaServiceHealth(OpUaConnectionServiceTest.java:74)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- testOpcUaServiceHealth -->
</testsuite> <!-- OPC UA Connection Service Tests -->
